README.Rmd

```diff
@@ -15,71 +15,125 @@ knitr::opts_chunk$set(
+| [IPUMS International](https://international.ipums.org/international/) | Microdata      | Census microdata covering over 100 countries, contemporary and historical                                                                                                                                                                               |
+| [IPUMS Time Use](https://timeuse.ipums.org/)                          | Microdata      | Time use microdata from the U.S. (1930-present) and thirteen other countries (1965-present)                                                                                                                                                             |
+| [IPUMS Health Surveys](https://healthsurveys.ipums.org/)              | Microdata      | Microdata from the U.S. [National Health Interview Survey (NHIS)](https://nhis.ipums.org/nhis/) (1963-present) and [Medical Expenditure Panel Survey (MEPS)](https://meps.ipums.org/meps/) (1996-present)                                               |
+| [IPUMS Global Health](https://globalhealth.ipums.org/)                | Microdata      | Health survey microdata for low- and middle-income countries, including harmonized data collections for [Demographic and Health Surveys (DHS)](https://www.idhsdata.org/) and [Performance Monitoring for Action (PMA)](https://pma.ipums.org/) surveys |
+| [IPUMS Higher Ed](https://highered.ipums.org/highered/)               | Microdata      | Survey microdata on the science and engineering workforce in the U.S. from 1993 to 2013                                                                                                                                                                 |
+| [IPUMS NHGIS](https://www.nhgis.org/)                                 | Aggregate Data | Tabular U.S. Census data and GIS boundary files (1790-present)                                                                                                                                                                                          |
+| [IPUMS IHGIS](https://ihgis.ipums.org/)                               | Aggregate Data | Tabular and GIS data from population, housing, and agricultural censuses around the world                                                                                                                                                               |
+
+## Installation
+
+To install the package from CRAN, use
 
-The ipumsr package can be installed by running the following command:
 ```{r, eval=FALSE}
 install.packages("ipumsr")
 ```
 
-Or, you can install the development version using the following commands:
-```{r, eval=FALSE}
-if (!require(devtools)) install.packages("devtools")
+To install the development version of the package, use
 
+```{r, eval=FALSE}
 devtools::install_github("ipums/ipumsr")
```

Comment:
Is there any reason we shouldn't change this `remotes::install_github()`?

Reply:
Not exactly. I think it actually lives in `remotes` but I have seen it referenced in both ways. I suppose we can just do what's most consistent with the language in the rest of ipumsr